doc/swss-schema.md

```diff
@@ -170,6 +170,24 @@ and reflects the LAG ports into the redis under: `LAG_TABLE:<team0>:port`
     family        = "IPv4" / "IPv6"  ; address family
 
 ---------------------------------------------
+###FDB_TABLE
+
+    ; Stores FDB entries which were inserted into SAI state manually
+    ; Notes:
+    ; - only unicast FDB entries supported
+    ; - only Vlan interfaces are supported
+    key           = FDB_TABLE:mac_address:"vlan"vlanid ; mac address will be inserted to FDB for the vlan interface
+    port          = ifName                ; interface where the entry is bound to
+    type          = "static" / "dynamic"  ; type of the entry
+
+    Example:
+    127.0.0.1:6379> hgetall FDB_TABLE:52:54:00:25:06:E9:Vlan2
```

Comment:
can we put vlan2 before the mac address? also, can we change the mac address to xx-xx-xx-xx-xx-xx, so that we do not confuse the separator?